doc/vim-go.txt

```diff
@@ -629,6 +668,19 @@ if			      "inside a function", select contents of a function,
 
 
 ===============================================================================
+FUNCTIONS                                                       *go-functions*
+
+There is only one function at the moment and it is neovim only.
```

Comment:
@fatih i'm actually still in high school. the assignments are so annoying only because programming in visual studio takes so long.